test/conftest.py

```diff
@@ -0,0 +1,10 @@
+import pytest
+from selenium import webdriver
+
+
+@pytest.fixture(scope="module")
+def driver():
+    driver = webdriver.Edge()
+    driver.maximize_window()
+    yield driver
```

Comment:
Buen uso de la fixture, pero te faltaria agregar un driver.quit() luego del yield para asegurarnos que se cierra al finalizar la prueba:
```suggestion
    yield driver
    driver.quit()
```